support-frontend/assets/pages/contributions-landing/contributionsLandingActions.js

```diff
@@ -464,14 +445,6 @@ const paymentAuthorisationHandlers: PaymentMatrix<(
       paymentAuthorisation: PaymentAuthorisation,
     ): Promise<PaymentResult> => {
       if (paymentAuthorisation.paymentMethod === Stripe) {
-        if (paymentAuthorisation.token) {
-          return dispatch(executeStripeOneOffPayment(
-            stripeChargeDataFromCheckoutAuthorisation(paymentAuthorisation, state),
-            (token: string) => dispatch(setGuestAccountCreationToken(token)),
-            (thankYouPageStage: ThankYouPageStage) => dispatch(setThankYouPageStage(thankYouPageStage)),
-          ));
-        }
-
         if (paymentAuthorisation.paymentMethodId) {
```

Comment:
as there's no longer two conditional branches inside the first `if` we could consider a rewrite as `(paymentAuthorisation.paymentMethod === Stripe && paymentAuthorisation.paymentMethodId)`

Reply:
Good spot, I've just added an extra level of error logging for this as the `paymentMethodId` should always be present at this point.